.gitignore

```diff
@@ -35,3 +35,5 @@ lib/generated_plugin_registrant.dart
 
 # Exceptions to above rules.
 !/packages/flutter_tools/test/data/dart_dependencies_test/**/.packages
+
+pubspeec.lock
```

Comment:
Let's not keep lock files in gitignore, we need to track library updates.

Reply:
So should I include it in the commit??

Reply:
yes please push the lock files also, we'll try to vc it later.

Reply:
Yep, I have pushed the lock file. Check it out